go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/IBM/ibmcloud-volume-file-vpc
 
-go 1.23.8
+go 1.23.10
+
+toolchain go1.24.4
 
 require (
 	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7
-	github.com/IBM/ibmcloud-volume-interface v1.2.12
```

Comment:
We are not doing any changes in interface revert this to latest version of interface